website/source/docs/concepts/policies.html.md

```diff
@@ -99,6 +111,31 @@ capabilities. These mappings are as follows:
 
   * `read` - `["read", "list"]`
 
+## Permissions
+
+Paths offer an optional `permissions` field for fine-grain control over the
+parameters and values a path is able to write. The capabilities associated with
+this path take precedence over permissions on parameters.
+
+  * `allowed_parameters` - Acts as a whitelist. Setting to "*" will allow a
+    parameter by any name to be changed.  Otherwise setting a key with an `[]`
+    value will allow changes to parameters with that name. Setting a key with a
+    populated value array will allow that parameter to only be set to one of the
+    values in the array. If keys exist in the `allowed_parameters` object all
+    keys not specified will be denied.  
```

Comment:
A bunch of things could be clarified here; rather than point out more in the review just give it a once-over and we can always update the website at any point if we want.